### DISCUSSION TITLE
Updating Guide with infographic: Managing Career Development

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#177] [Community Proposals]
I've read through one of the TODO Guides "Managing Career Development within OSPOs" and came up with the idea of adding an infographic (attached below) that summarises the guide's takeaways in just one page. 

I'd like to ask the community for feedback and assistance. Maybe we can use better wording or reduce content. Can you please leave your comments via this discussion thread? I will use this to create the final version of this infographic and submit a PR to the [TODO Website](https://github.com/todogroup/todogroup.org)

Also, if this format works, we could apply a similar design to each of the existing TODO guides. I'm sure some of them can be updated too 😉

* This is the [PDF](https://github.com/todogroup/ospology/files/9505034/managing.career.development.within.OSPOs.infographic-1-2.pdf)

* This is the png:
![managing career development within OSPOs infographic(1)(1)](https://user-images.githubusercontent.com/43671777/188850475-a88b5aaf-f6d3-4423-8724-f0f97a88c6f8.png)

**Comment by caniszczyk** [comment]
+1 to having some type of one pager for OSPOs to hand out at conferences and other places :)
